tsconfig.json

```diff
@@ -0,0 +1,102 @@
+{
```

Comment:
Can we remove the commented out config options

Reply:
✅Done